ffs/scheduler/scheduler.go

```diff
@@ -42,7 +42,8 @@ type Scheduler struct {
 	cis *cistore.Store
```

Comment:
The change in the scheduler runner was the following.

There're two events that should trigger queued Job evaluation for run:
- When a new Job is queued.
- When a Executing Job finished.

The last point is important for two cases:
- If a queued job that's runnable couldn't get into _Executing_, is because the runner rate limiting is full. So it needs to be reevaluated for a slot whenever someone finished.
- Even if the rate-limiting isn' t full, a queued Job could not be runnable. That's the case if there's another Job fo the same Cid running. If that's the case, it can only be run when that executing Job finishes... hence the finishing triggering event.